pkg/store/copr/batch_coprocessor.go

```diff
@@ -1483,7 +1483,7 @@ func buildBatchCopTasksConsistentHashForPD(bo *backoff.Backoffer,
 		splitKeyStart := time.Now()
 		for i, ranges := range rangesForEachPhysicalTable {
 			rangesLen += ranges.Len()
-			locations, err := cache.SplitKeyRangesByLocationsWithoutBuckets(bo, ranges, UnspecifiedLimit)
+			locations, err := cache.SplitKeyRangesByLocations(bo, ranges, UnspecifiedLimit, false, false)
```

Comment:
@windtalker PTAL this file, I unify `SplitKeyRangesByLocationsWithBuckets` and `SplitKeyRangesByLocationsWithoutBuckets`. For `batch_coprocessor`, region without leader peer will also be returned(though this case is impossible in our expection).

Reply:
LGTM